Orange/tests/test_table.py

```diff
@@ -2195,6 +2195,12 @@ class _ExtendedTable(data.Table):
         self.assertIsInstance(data_file, _ExtendedTable)
         self.assertIsInstance(data_url, _ExtendedTable)
 
+    def test_str(self):
+        try:
```

Comment:
If I add this test (without changing the problematic code) to master, all tests pass. Could you add a test that fails until the error is fixed?